src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -559,8 +559,11 @@ export default class DefaultAudioVideoController implements AudioVideoController
         this.logger.info('canceled retry');
       }
     );
-
     if (!willRetry) {
+      errorMessage &&
+        this.logger.error(
+          `failure message of ${MeetingSessionStatusCode[status.statusCode()]}: ${errorMessage}`
```

Comment:
Can we rephrase the start of this error message? Something like 

`reconnection failed with status code ${MeetingSessionStatusCode[status.statusCode()]}: ${errorMessage}`

Reply:
Yeah sure, we can rephrase the error message, but I'm not sure if `reconnection failed..` is accurate here as the error message could be from actionConnect or actionUpdate failure